bus/src/main/java/org/hawkular/metrics/component/publish/InsertedDataSubscriber.java

```diff
@@ -64,8 +68,21 @@
 
     private Subscription subscription;
 
+    @Inject
+    private PublishCommandTable publish;
+
+    @Inject
+    @Configurable
+    @ConfigurationProperty(METRICS_PUBLISH_PERIOD)
+    private String publishPeriodProperty;
+    private int publishPeriod;
+
     public void onMetricsServiceReady(@Observes @ServiceReady ServiceReadyEvent event) {
-        Observable<List<Metric<?>>> events = event.getInsertedData().buffer(50, TimeUnit.MILLISECONDS, 100)
+        publishPeriod = getPublishPeriod();
+        Observable<List<Metric<?>>> events = event.getInsertedData()
+                .filter(m -> m.getType() != MetricType.STRING)
+                .filter(m -> publish.isPublished(m.getMetricId()))
```

Comment:
It is filtering by the MetricId type 
https://github.com/hawkular/hawkular-metrics/pull/574/files#diff-c8641cf1ba58303ba15fba81de0f0c1aR42
Which is supplied from alerting in the proposal
https://github.com/hawkular/hawkular-alerts/pull/197/files#diff-d64b0cb70fe108f47cd45e05615cec19R160

If I haven't introduced any bug I think the design is valid.